docs/RationaleHW-SV.md

```diff
@@ -388,6 +388,32 @@ still many things in SystemVerilog that we cannot currently express in the SV
 dialect, this design makes it easy to incrementally build out new capabilities
 over time.
 
+#### Verbatim op
+The verbatim operation produces a typed value expressed by a string of
+SystemVerilog.  This can be used to access macros and other values that are
+only sensible as Verilog text. There are three kinds of verbatim operation, 
+ 1. VerbatimOp(`sv.verbatim`, the statement form
+ 2. VerbatimExprOp(`sv.verbatim.expr`), the expression form.
+ 3. VerbatimExprSEOp(`sv.verbatim.expr.se`), the expression form.
+
+For the verbatim expression form, the text string is expected to have the
+highest precedence, so parentheses is required if it isn't a single token.
+`sv.verbatim` and `sv.verbatim.expr` are assumed to not have side effects, 
```

Comment:
Minor but important thing: sv.verbatim has side effects, it is only sv.verbatim.expr that doesn't.